test/e2e/network/kube_proxy.go

```diff
@@ -338,3 +374,22 @@ var _ = SIGDescribe("Network", func() {
 		}
 	})
 })
+
+// fullIPv6 returns a string with the IP representation
```

Comment:
should this be a utility method?

Reply:
I didn't see it used anywhere else, it seems that only conntrack is expanding the IPv6 addresses when writing to a file.
I don't have any preference honestly, just thought this was the simplest way